### ISSUE TITLE
documentation for custom JVM?

### ISSUE THREAD

**Issue by williamj77** [javapathfinder/jpf-core#502]
Hello,

Is there documentation for the Pathfinder custom JVM?

williamj

**Comment by cyrille-artho** [comment]
No, unfortunately not; we currently focus on the JVM. If someone adds support for a different VM, then that work could also be illustrative and be used as a starting point for documentation.
